packages/discord.js/typings/index.d.ts

```diff
@@ -3907,7 +3907,19 @@ export interface ApplicationCommandSubCommandData extends Omit<BaseApplicationCo
 
 export interface ApplicationCommandSubCommand extends Omit<BaseApplicationCommandOptionsData, 'required'> {
   type: ApplicationCommandOptionType.Subcommand;
-  options?: (ApplicationCommandChoicesOption | ApplicationCommandNonOptions | ApplicationCommandChannelOption)[];
+  options?: (    
+    | ApplicationCommandNonOptionsData
+    | ApplicationCommandChannelOptionData
+    | ApplicationCommandChoicesData
+    | ApplicationCommandAutocompleteOption
+    | ApplicationCommandNumericOptionData
+    | ApplicationCommandStringOptionData
+    | ApplicationCommandRoleOptionData
+    | ApplicationCommandUserOptionData
+    | ApplicationCommandMentionableOptionData
+    | ApplicationCommandBooleanOptionData
+    | ApplicationCommandAttachmentOption
+  )[];
 }
```

Comment:
Fairly sure this isn't correct and needs to be using the types that aren't suffixed with "data".

Reply:
Thanks I changed it